lib/spack/spack/filesystem_view.py

```diff
@@ -96,9 +97,9 @@ def view_copy(
         prefix_to_projection[spack.store.STORE.layout.root] = view._root
 
         # This is vestigial code for the *old* location of sbang.
-        prefix_to_projection[f"#!/bin/bash {spack.paths.spack_root}/bin/sbang"] = (
-            sbang.sbang_shebang_line()
-        )
+        prefix_to_projection[
+            f"#!/bin/bash {spack.paths.spack_root}/bin/sbang"
+        ] = sbang.sbang_shebang_line()
```

Comment:
spackbot broke the format on these lines :disappointed: Maybe we are using different versions of `black` for the bot and style checks?